Algorithm/QCAlgorithm.cs

```diff
@@ -544,31 +544,16 @@ public virtual void PostInitialize()
             // if the benchmark hasn't been set yet, set it
             if (Benchmark == null)
             {
-                if (_benchmarkSymbol != null)
+                if (_benchmarkSymbol == null)
                 {
-                    // if the requested benchmark symbol wasn't already added, then add it now
-                    // we do a simple compare here for simplicity, also it avoids confusion over
-                    // the desired market.
-                    Security security;
-                    if (!Securities.TryGetValue(_benchmarkSymbol, out security))
-                    {
-                        // add the security as an internal feed so the algorithm doesn't receive the data
-                        security = CreateBenchmarkSecurity();
-                    }
-
-                    // just return the current price
-                    Benchmark = new SecurityBenchmark(security);
-                }
-                else
-                {
-                    var start = StartDate;
-                    var startingCapital = Portfolio.TotalPortfolioValue;
-                    Benchmark = new FuncBenchmark(dt =>
-                    {
-                        var years = (dt - start).TotalDays / 365.25;
-                        return startingCapital * (decimal) Math.Exp(0.02 * years);
-                    });
+                    _benchmarkSymbol = QuantConnect.Symbol.Create("SPY", SecurityType.Equity, Market.USA);
                 }
+
+                var security = Securities.CreateSecurity(_benchmarkSymbol,
```

Comment:
Do we want to check the `Securities` collection to see if it already exists?

Reply:
My intention was to avoid using the same security instance used by the algorithm, this allows us to avoid multiple issues e.g. `Security.Leverage` being wrong, price updates https://github.com/QuantConnect/Lean/blob/master/Engine/AlgorithmManager.cs#L338

Reply:
OK, I see the motive, I guess I'm just surprised that this works. The subscription in the data feed would need to be linked to _this_ security object, right? In other words, how does this `security` instance receive pricing updates?